src/lib/core/components/computations/ComputationInstance.tsx

```diff
@@ -105,32 +106,17 @@ interface AppTitleProps {
   condensed: boolean;
 }
 
-// We expect two different types of app titles: one in /visualizations that labels each app's row
-// and one just below the navigation when we're working on a viz within an app. The former
-// is the "condensed" version. May make sense to break into two components when
-// further styling is applied?
 function AppTitle(props: AppTitleProps) {
   const { computation, condensed } = props;
-  const splitDisplayName = computation.displayName
-    ? computation.displayName.split('&;&')
-    : '';
+  const plugin = plugins[computation.descriptor?.type];
+  const ConfigDescription = plugin.configurationDescriptionComponent;
```

Comment:
It occurs to me that this is unsafe, as `plugin` could potentially be `undefined`. I think the type of `plugins` needs to be updated as such. I think we should also throw an error here, if `plugin` is undefined. Perhaps an assert function?